spec/api/houdini/nonprofit_spec.rb

```diff
@@ -92,7 +92,7 @@ def h(h = {})
 
       expect_any_instance_of(SlugNonprofitNamingAlgorithm).to receive(:create_copy_name).and_raise(UnableToCreateNameCopyError.new)
 
-      xhr :post, '/api/v1/nonprofit', input
+      post '/api/v1/nonprofit', params:input, xhr: true
```

Comment:
xhr and the use of positional params in request specs is deprecated so this corrected that.